### PR TITLE
Added Example to Upload Images using Ruby SDK 

### LINKED ISSUE #68: [Example] Ruby - Uploading images
Create an example that shows how to upload images using the Cloudinary Ruby SDK

If it makes sense, this can be a script that can be executed via the terminal.

The example can be the same as the Node.js SDK example, but using the Ruby SDK.

https://github.com/cloudinary-community/cloudinary-examples/tree/main/examples/node-image-upload

### PR DESCRIPTION
**Pull request by aazam-gh**
Fix #68 

Similar to node sdk example to upload images to cloudinary. An example has been made for uploading images using Cloudinary Ruby SDK.
README file describes simple steps to get started on adding cloudinary to your Gemfile

### CODE REVIEW

File: examples/ruby-image-upload/Gemfile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+source 'https://rubygems.org'
+
+gem 'cloudinary'
+group :development do
+  gem 'dotenv-rails'
```
Comment (colbyfayock) [inline examples/ruby-image-upload/Gemfile:5]: curious about this one, it imports in the ruby file as `dotenv`, is that accurate?

File: examples/ruby-image-upload/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+* Create an environment variable file `.env` and inside add:
+```
+CLOUDINARY_CLOUD_NAME="<Your Cloud Name>"
+```
+
+* Install the project dependencies with:
+
+```
+gem install
```
Comment (colbyfayock) [inline examples/ruby-image-upload/README.md:38]: `gem install` doesnt seem to install all the dependencies, looks like you need to use bundler?

https://stackoverflow.com/a/44406252/844780

File: examples/ruby-image-upload/README.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+```
+image = './path/to/image.jpg' # Can be a local file path or a remote URL
+result = Cloudinary::Uploader.upload(image)
+```
+
+## 🚀 Get Started with This Example
+
+* Create an environment variable file `.env` and inside add:
+```
+CLOUDINARY_CLOUD_NAME="<Your Cloud Name>"
```
Comment (colbyfayock) [inline examples/ruby-image-upload/README.md:32]: You need to add the API KEY and API SECRET to the instructions, not just the cloud name, for getting the local example running

**Comment by colbyfayock** [comment]
hey @aazam-gh got it running and worked great! have a few comments here first before we merge it in. let me know what you think

File: examples/ruby-image-upload/README.md (hunk 2 above)
Comment (aazam-gh) [reply examples/ruby-image-upload/README.md:38]: Yes the user needs to have bundler already installed for gem install to work.

File: examples/ruby-image-upload/README.md (hunk 3 above)
Comment (aazam-gh) [reply examples/ruby-image-upload/README.md:32]: Noted. I'll add the two as well

File: examples/ruby-image-upload/Gemfile (hunk 1 above)
Comment (aazam-gh) [reply examples/ruby-image-upload/Gemfile:5]: Yes this will allow us to use env

**Comment by aazam-gh** [comment]
@colbyfayock I have made some changes based on the reviews :)

**Comment by colbyfayock** [comment]
looks great, thanks @aazam-gh

**Comment by colbyfayock** [comment]
@aazam-gh congrats on the merged PR! since it was merged during Hacktoberfest, you're eligible for some free swag. Please email [hacktoberfest@cloudinary.com](mailto:hacktoberfest@cloudinary.com) with your Github username and a link to this PR where we'll follow up with you there.

https://cloudinary.com/blog/hacktoberfest-celebrate-open-source-sdks

**Comment by colbyfayock** [comment]
@all-contributors please add @aazam-gh for code

**Comment by aazam-gh** [comment]
@colbyfayock Thanks for merging. I have sent an email :D

**Pull request merged**